### PR TITLE
fix(acl): add the placeholders manual for build-in acl file

### PR DESCRIPTION
**Pull request by hjianbo**
For https://github.com/emqx/emqx/issues/4174

### CODE REVIEW

File: en_US/advanced/acl-file.md [hunk 1]
```diff
@@ -36,23 +36,23 @@ The built-in ACL is the lowest priority rule table. If it is not hit after all t
 The rules file is described in Erlang syntax:
 
 ```erlang
-%% Allow "dashboard" users to subscribe to "$ SYS / #" topics
+%% Allow "dashboard" users to subscribe to "$ SYS/#" topics
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]: why the space after `$`?

File: en_US/advanced/acl-file.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ The syntax rules of `acl.conf` are included in the comments at the top. Those fa
     * `subscribe`：The rule applies to SUBSCRIBE operations
     * `pubsub`：The rule applies to both PUBLISH and SUBSCRIBE operations
 - The fourth position of the tuple means the list of topics restricted by the rule. The content is given in the form of an array. For example:
-    * `"$SYS/#"`：a **Topic Filter** which means that the rule can hit topics that match `$ SYS / #`; for example, it can hit "$SYS/#" and "$SYS/a/b/c"
+    * `"$SYS/#"`：a **Topic Filter** which means that the rule can hit topics that match `$SYS/#`; for example, it can hit "$SYS/#" and "$SYS/a/b/c"
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]:
```suggestion
    * `"$SYS/#"`：a **Topic Filter** which means that the rule is applied to topics that match `$SYS/#`; for example rules created for "$SYS/#" applies to publish/subscribe actions for topic "$SYS/a/b/c"
```

File: en_US/advanced/acl-file.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ The syntax rules of `acl.conf` are included in the comments at the top. Those fa
...
     * `{eq, "#"}`：It indicates full equivalence of characters. The rule can only hit strings with the topic `#`. It cannot hit `/a/b/c`, etc.
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]:
```suggestion
    * `{eq, "#"}`：It indicates full equivalence of characters. The rule is only applied for topic `#` but not for `/a/b/c`, etc.
```

File: en_US/advanced/acl-file.md [hunk 4]
```diff
@@ -87,10 +87,26 @@ The syntax rules of `acl.conf` are included in the comments at the top. Those fa
...
 ```bash
-./bin/emqx_ctl acl reload
+./bin/emqx_ctl modules reload emqx_mod_acl_internal
 ```
 
+## Placeholders
+
+The built-in `acl.conf` supports only the following placeholders in the subject's field (the 4th position of the tuple).
+
+- `%c`: Indicates the Client ID, which will be replaced with the actual client ID when the rule takes effect.
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]:
```suggestion
- `%c`: For Client ID, which is replaced by the client ID when the rule takes effect.
```

File: en_US/advanced/acl-file.md [hunk 5]
```diff
@@ -87,10 +87,26 @@ The syntax rules of `acl.conf` are included in the comments at the top. Those fa
...
+- `%u`: Indicates the client username, which will be replaced with the actual client username when the rule takes effect.
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]:
```suggestion
- `%u`: For username, which is replaced by the client's username when the rule takes effect.
```

File: en_US/advanced/acl-file.md [hunk 6]
```diff
@@ -87,10 +87,26 @@ The syntax rules of `acl.conf` are included in the comments at the top. Those fa
...
+
+E.g:
+
+```erlang
+{allow, all, pubsub, ["sensor/%c/ctrl"]}.
+```
+
+Means that clients with ID light are **Allowed** to **Subscribe and Publish** to the `sensor/light/ctrl` topic.
```
Comment (zmstone) [inline en_US/advanced/acl-file.md]:
```suggestion
Means that a client with ID 'light' is **Allowed** to **Subscribe and Publish** to the `sensor/light/ctrl` topic.
```

File: en_US/advanced/acl-file.md (hunk 1 above)
Comment (hjianbo) [reply en_US/advanced/acl-file.md]: Nice eyes!

**Comment by hjianbo** [comment]
Cool @zmstone Thanks for your correction

**Review by zmstone — APPROVED** [review]

**Pull request merged**